sdk/extensions/Microsoft.Extensions.Azure/src/Internal/ClientFactory.cs

```diff
@@ -228,7 +228,7 @@ internal static TokenCredential CreateCredential(IConfiguration configuration)
 
             if (!string.IsNullOrWhiteSpace(objectId))
             {
-                throw new ArgumentException("Managed identity 'objectId' is only supported when the credential type is 'managedidentity'.");
+                throw new ArgumentException("'managedIdentityObjectId' is only supported when the credential type is 'managedidentity'.");
```

Comment:
should we add a changelog entry for this? Normally I'd say it's not significant enough, but if we want to release this, we'll need something to differentiate it from the one we just released today.

Reply:
I don't think it's necessary. I'm okay with not shipping a new version just for this change. Let's wait until there's something else that justifies a new release. Sound good?

Reply:
Sounds good to me!